LBHackney-IT/lbh-social-care
# Feature/add address lookup

**What**  
Added `AddressLookup` component

<img width="905" alt="Screenshot 2020-10-28 at 16 37 15" src="https://user-images.githubusercontent.com/435399/97467381-6c471380-1944-11eb-94bd-1befe9bfc856.png">

with manual input
<img width="981" alt="Screenshot 2020-10-28 at 16 37 24" src="https://user-images.githubusercontent.com/435399/97467388-6e10d700-1944-11eb-9036-72727edd59c4.png">

and postcode lookup
<img width="849" alt="Screenshot 2020-10-28 at 16 38 09" src="https://user-images.githubusercontent.com/435399/97467424-79640280-1944-11eb-8b1d-8561c7617889.png">

**Extras**
Added storybook proxy for `/api`

**How to test it**
- checkout this branch
- install dependencies `yarn`
- spin up the server `yarn dev`
- spin up storybook `yarn storybook`
- go to http://localhost:6006/?path=/story/form-addresslookup--default
- check that the component works as expected
